aspnetcore/fundamentals/middleware/index/snapshot/StartupAll3.cs

```diff
@@ -53,6 +53,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
             app.UseAuthorization();
             // app.UseSession();
+            // app.UseResponseCompression();
```

Comment:
Might want to compress before caching? @Tratcher

Reply:
This is fine. Caching and compression ordering are scenario specific, there's multiple valid orderings. Yes, you can compress and then cache, but then you might need to store multiple compressed representations. I've also seen these combine with static files to allow you to cache compressed static files.
UseResponseCaching
UseResponseCompression
UseStaticFiles